src/segment-loader.js

```diff
@@ -2492,6 +2491,8 @@ export default class SegmentLoader extends videojs.EventTarget {
       return;
     }
 
+    this.trigger('appendsdone');
```

Comment:
Don't report aborted appends as appends

Reply:
Do you know in what instances we will abort appends? It seems like it would only happen on dispose of the source buffers, otherwise appends will continue even if the current segment loader process is aborted.

Reply:
I think it's the request itself in this case that is aborted, rather than an actual append. I was seeing it with aggressive network throttling and changing renditions.

Reply:
Might be worth us adding a test for this case.